spec/models/candidate_interface/personal_details_form_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+RSpec.describe CandidateInterface::PersonalDetailsForm, type: :model do
+  describe '#name' do
+    it 'concatenates the first name and last name ' do
+      personal_details = FactoryBot.build(:personal_details_form, first_name: 'Bruce', last_name: 'Wayne')
```

Comment:
Would it be better to use the class here directly, instead of going via the factory?